.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - "stable"
+install: npm install
+services:
+  - postgresql
+cache:
+  directories:
+    - "node_modules"
+before_script:
+  - psql -c 'create database travis_ci_test;' -U postgres
+script: npm run test
```

Comment:
`npm test` works aswell

Reply:
yes thats true. I tend to just use 'run' most times. I will change it now